modules/ding_provider/tests/ding_provider_implementation.test

```diff
@@ -622,4 +608,17 @@ abstract class DingSearchProviderImplementationTestCase extends DingProviderImpl
     $this->assertEqual($query_exclude->getMaterialFilter(), ['123']);
     $this->assertFalse($query_exclude->isMaterialFilterInclude());
   }
+
+  /**
+   * Test that search strings are quoted.
+   */
+  public function testSearchQuoting() {
+    // Connie does a simple quoting of quotes.
```

Comment:
Doesn't make much sense to me.
Connie escapes quotes?

Reply:
Fixed